be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/Validator.scala

```diff
@@ -51,6 +69,7 @@ object Validator {
   }
 
   // contains the relative path to our "main" query JsonSchema file
+  //FIXME: does not work apparently
```

Comment:
Fixed in PR#583 commit '66d492f28a788fe6954c5230de778c5209221370'

Reply:
indeed, I forgot to remove the fixme after I pulled